internal/rpcserver/rpcserver.go

```diff
@@ -58,7 +58,7 @@ import (
 // API version constants
 const (
 	jsonrpcSemverMajor = 7
-	jsonrpcSemverMinor = 0
+	jsonrpcSemverMinor = 1
```

Comment:
This major has already been bumped since the last release, so I don't really see any reason to bump the minor here again.

Reply:
Not bumping.